test/e2e/kubectl.go

```diff
@@ -110,17 +110,71 @@ var (
 	//
 	// TODO(ihmccreery): remove once we don't care about v1.1 anymore, (tentatively in v1.4).
 	podProbeParametersVersion = version.MustParse("v1.2.0-alpha.4")
+
+	kubectl_guestbookController  = "kubectl_guestbookController.json"
+	kubectl_guestbookSvc         = "kubectl_guestbookService.json"
+	kubectl_updateDemonautilusRC = "kubectl_updateDemonautilusRC.json"
+	kubectl_updateDemoKittyRC    = "kubectl_updateDemoKitty.json"
+	kubectl_userguidePod         = "kubectl_userguidePod.json"
 )
 
+// This map allows us to pass Conformance tests without relying on repo-root
+var kubectlExampleFiles = map[string]string{
+	// 1 pod instead of 3 for fast testing.
+	kubectl_guestbookController: `{"kind":"ReplicationController",
+			"apiVersion":"v1",
+			"metadata":{"name":"guestbook","labels":{"app":"guestbook"}},
+			"spec":{"replicas":1,"selector":{"app":"guestbook"},"template":{"metadata":{"labels":{"app":"guestbook"}},
+			"spec":{"containers":[{"name":"guestbook","image":"gcr.io/google_containers/guestbook:v3","ports":[{"name":"http-server","containerPort":3000}]}]}}}}`,
+	kubectl_guestbookSvc: `{
+			"kind":"Service",
+			"apiVersion":"v1",
+			"metadata":{"name":"guestbook","labels":{"app":"guestbook"}},
+			"spec":{"ports":[{"port":3000,"targetPort":"http-server"}],"selector":{"app":"guestbook"},"type":"LoadBalancer"}}`,
+	kubectl_updateDemonautilusRC: `{"apiVersion":"v1","kind":"ReplicationController",
+				"metadata":{"name":"update-demo-nautilus"},
+				"spec":{"replicas":2,"selector":{"name":"update-demo","version":"nautilus"},
+				"template":{
+					"metadata":{"labels":{"name":"update-demo","version":"nautilus"}},
+					"spec":{"containers":[{"image":"gcr.io/google_containers/update-demo:nautilus",
+						"name":"update-demo","ports":[{"containerPort":80, "protocol":"TCP"}]}]}}}}`,
+	kubectl_updateDemoKittyRC: `{"apiVersion":"v1","kind":"ReplicationController",
+				"metadata":{"name":"update-demo-kitten"},
+				"spec":{"selector":{"name":"update-demo","version":"kitten"},
+				"template":{"metadata":{"labels":{"name":"update-demo","version":"kitten"}},
+				"spec":{"containers":[{"image":"gcr.io/google_containers/update-demo:kitten",
+				"name":"update-demo","ports":[{"containerPort":80,"protocol":"TCP"}]}]}}}}`,
+	kubectl_userguidePod: `{"apiVersion":"v1",
+			"kind":"Pod",
+			"metadata":{"labels":{"app":"nginx"},"name":"nginx"},
+			"spec":{"containers":[{"image":"nginx","name":"nginx","ports":[{"containerPort":80}]}]}}`,
+}
+
+var writeExampleFiles = func() {
+	for fileName, contents := range kubectlExampleFiles {
```

Comment:
i definetely agree its silly to write files, and yes, we can do this, just wanna make sure everyone is ok with this.  since it really changes the test coverage IMO.